### ISSUE TITLE
Support `--side-by-side` switch

### ISSUE THREAD

**Issue by rpdelaney** [mookid/diffr#46]
I'm not getting diffr highlighting when using `colordiff -y foo bar | diffr`. Would be neat to support that also.

**Comment by mookid** [comment]
I assume that colordiff -y makes the splitting.

I think that the parsing would be too tedious to be honest. Another possibility would be to implement the side-by-side rendering in diffr, but it is out of reach given the capabilities of the terminal currently used.  

Anyway, I would be very curious to see anyone giving that a try :)

**Comment by rpdelaney** [comment]
Yeah `-y` is short for `--side-by-side`. `colordiff` isn't necessary, that's just muscle memory on my part. GNU `diff` is the same.

I could give it a shot except rust isn't (yet) my jam.
